text/case.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+import { capitalizeWord, split } from "./_util.ts";
+
+/**
+ * Converts a input into camelCase
```

Comment:
```suggestion
 * Converts a string into camelCase.
```
Ditto for other instances.